python/python/glide/async_commands/cluster_commands.py

```diff
@@ -521,40 +528,40 @@ async def sort(
         See https://valkey.io/commands/sort for more details.
 
         Args:
-            key (str): The key of the list, set, or sorted set to be sorted.
+            key (TEncodable): The key of the list, set, or sorted set to be sorted.
             limit (Optional[Limit]): Limiting the range of the query by setting offset and result count. See `Limit` class for more information.
             order (Optional[OrderBy]): Specifies the order to sort the elements.
                 Can be `OrderBy.ASC` (ascending) or `OrderBy.DESC` (descending).
             alpha (Optional[bool]): When `True`, sorts elements lexicographically. When `False` (default), sorts elements numerically.
                 Use this when the list, set, or sorted set contains string values that cannot be converted into double precision floating point numbers.
 
         Returns:
-            List[str]: A list of sorted elements.
+            List[bytes]: A list of sorted elements.
 
         Examples:
-            >>> await client.lpush("mylist", '3', '1', '2')
-            >>> await client.sort("mylist")
-            ['1', '2', '3']
+            >>> await client.lpush(b"mylist", b'3', b'1', b'2')
+            >>> await client.sort(b"mylist")
+            [b'1', b'2', b'3']
 
-            >>> await client.sort("mylist", order=OrderBy.DESC)
-            ['3', '2', '1']
+            >>> await client.sort(b"mylist", order=OrderBy.DESC)
+            [b'3', b'2', b'1']
 
-            >>> await client.lpush("mylist", '2', '1', '2', '3', '3', '1')
-            >>> await client.sort("mylist", limit=Limit(2, 3))
-            ['1', '2', '2']
+            >>> await client.lpush(b"mylist", b'2', b'1', b'2', b'3', b'3', b'1')
+            >>> await client.sort(b"mylist", limit=Limit(2, 3))
+            [b'2', b'2', b'3']
 
-            >>> await client.lpush("mylist", "a", "b", "c", "d")
-            >>> await client.sort("mylist", limit=Limit(2, 2), order=OrderBy.DESC, alpha=True)
-            ['b', 'a']
+            >>> await client.lpush(b"mylist", b"a", b"b", b"c", b"d")
```

Comment:
```suggestion
            >>> await client.lpush(b"mylist", [b"a", b"b", b"c", b"d"])
```
this is the correct syntax